tests/testthat/helper-setup.R

```diff
@@ -0,0 +1,235 @@
+
+# Code that needs to run once, before a suite of tests is run.
+# Here, "suite of tests" might also mean "a single test" interactively.
+renv_tests_setup <- function(envir = parent.frame()) {
+
```

Comment:
I tried to separate "setup" functions (`renv_setup_*`) from "fixture" functions (`renv_scope_*`), so that the "setup" functions only ever need to be called once, currently in `helper-zzz.R` or `setup.R`.

So, by design, I don't think this function can be called multiple times? If it is, that's programmer error, I think.

Reply:
Ok, makes sense.